server_tests/test/clientTests.js

```diff
@@ -773,8 +800,181 @@ describe('RUST API TEST SUITE', function () {
     });
   });
 
+  // LIST FILES (OSC2.0)
+  describe('Testing /osc/commands/execute camera.listFiles endpoint', function () {
+    if (!isOSC2) {
+      return this.skip()
+    }
+    var expectedImageCount
+    var expectedVideoCount
+    var totalEntryCount
+
+    before(function () {
+      // delete exisiting files on SD card, if any
+      return Utility.deleteAllImages() //should have a new utility deleteAll() or something
+      // take a bunch of photos and videos
+      .then(function onSuccess () {
+        for (let i = 0; i < 3; i++) {
+        return testClient.takepicture
+        expectedImageCount++
+        totalEntryCount++
+        }
+      }).then(validate.done(res.body, schema.names.commandTakePicture))
+      .then(testClient.bublCaptureVideo
+      .then(function onSuccess () {
+        expectedVideoCount++
+        totalEntryCount++
+        validate.done(res.body, schema.names.commandBublCaptureVideo)
+      })
+    })
+
+    after(function () {
+      // delete all files on SD card
+      return Utility.restoreDefaultOptions(defaultOptionsFile)
+      .then(Utility.deleteAllImages)
```

Comment:
I think not any more since I added the last test. Originally just wanted to delete all files on the SD card at the end of the test suite